src/styles/cards.scss

```diff
@@ -0,0 +1,143 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(4,200px);
+  grid-gap: 46px 48px;
+  justify-content: center;
+  padding-top: 50px;
+  padding-bottom: 72px;
+}
+
+.card {
+  border: 1px solid $border-color;
+  width: 200px;
+  height: 408px;
+  display: flex;
+  flex-direction: column;
+  background-color: $white-color;
+  border-radius: 5px;
+  justify-content: space-between;
+  transition: transform 300ms;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover &__product-name {
+    color: $title-color;
+  }
+
+  &__image {
+    margin: 19px;
+    margin-top: 32px;
+    display: block;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-name {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    padding: 21px 16px 0;
+    color: $black-color;
+  }
+
+  &__product-code {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin: 16px;
+    margin-top: 4px;
+    margin-bottom: 16px;
+    color: $gray-color;
+  }
```

Comment:
Fix this according [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md) hints. Fix all cases
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/24671564-bfcf-409b-871f-ff5d8bd7caab)